wdl-analysis/src/rules.rs

```diff
@@ -38,6 +38,11 @@ pub trait Rule: Send + Sync {
 
     /// Gets the severity of the rule.
     fn severity(&self) -> Severity;
+
+    /// Gets the related rules ids for this rule.
+    fn related_rules(&self) -> &'static [&'static str] {
+        &[]
+    }
```

Comment:
I'm not sure about adding it here but we want to support all rules

Reply:
I think this should be removed, but I don't feel super strongly about it. Let's leave it for now, and anyone else reviewing can weigh in with their thoughts 🤷‍♀️

Reply:
I've thought about it some more and I'm now more convinced this should be removed. I don't think it adds anything to analysis rules, and in practice is just a needless complication. It's fine for `related_rules()` to be exclusive to wdl lint rules.